---
title: "# Connectomics
"
collection: notes
permalink: /notes/connectomics
date: 2016-06-01

---

# Connectomics

2016-06-01

Connectomics is the subfield of neuroscience related with the production and study of connectomes by assembling and analyzing connectome data sets. Connectome is a map description of neural connections in the brain. More broadly, a connectome would include the mapping of all neural connections within an organism's nervous system.

May range in scale from a detailed map of the full set of neurons and synapses within part or all of the nervous system of an organism to a macro scale description of the functional and structural connectivity between all cortical areas and subcortical structures. The term "connectome" is used primarily in scientific efforts to capture, map, and understand the organization of neural interactions within the brain.

Research has successfully constructed the full connectome of one animal: the roundworm C. elegans (White et al., 1986, Varshney et al., 2011). Partial connectomes of a mouse retina and mouse primary visual cortex have also been successfully constructed. Bock et al.'s complete 12 TB data set is publicly available at Open Connectome Project.

The ultimate goal of connectomics is to map the human brain. This effort is pursued by the Human Connectome Project, sponsored by the National Institutes of Health, whose focus is to build a network map of the human brain in healthy, living adults.

In 2005, Dr. Olaf Sporns at Indiana University and Dr. Patric Hagmann at Lausanne University Hospital independently and simultaneously suggested the term "connectome" to refer to a map of the neural connections within the brain. This term was directly inspired by the ongoing effort to sequence the human genetic code- to build a genome.


Current non-invasive imaging techniques cannot capture the brain's activity on a neuron-by-neuron level. Mapping the connectome at the cellular level in vertebrates currently requires post-mortem microscopic analysis of limited portions of brain tissue. Non-optical techniques that rely on high-throughput DNA sequencing have been proposed recently by Tony Zador (CSHL).

Traditional histological circuit-mapping approaches rely on imaging and include light-microscopic techniques for cell staining, injection of labeling agents for tract tracing, or reconstruction of serially sectioned tissue blocks via electron microscopy (EM). Each of these classical approaches has specific drawbacks when it comes to deployment for connectomics. The staining of single cells, e.g. with the Golgi stain, to trace cellular processes and connectivity suffers from the limited resolution of light-microscopy as well as difficulties in capturing long-range projections. Tract tracing, often described as the "gold standard" of neuroanatomy for detecting long-range pathways across the brain, generally only allows the tracing of fairly large cell populations and single axonal pathways. EM reconstruction was successfully used for the compilation of the C. elegans connectome (White et al., 1986). However, applications to larger tissue blocks of entire nervous systems have traditionally had difficulty with projections that span longer distances.

Recent advances in mapping neural connectivity at the cellular level offer significant new hope for overcoming the limitations of classical techniques and for compiling cellular connectome data sets (Livet et al., 2007; Lichtman et al., 2008). Using Brainbow, a combinatorial color labeling method based on the stochastic expression of several fluorescent proteins, Lichtman and colleagues were able to mark individual neurons with one of over 100 distinct colors. The labeling of individual neurons with a distinguishable hue then allows the tracing and reconstruction of their cellular structure including long processes within a block of tissue.

In March 2011, the journal Nature published a pair of articles on micro-connectomes: Bock et al. and Briggman et al. In both articles, the authors first characterized the functional properties of a small subset of cells, and then manually traced a subset of the processes emanating from those cells to obtain a partial subgraph. In alignment with the principles of open-science, the authors of Bock et al. (2011) have released their data for public access. The full resolution 12TB dataset from Bock et al. is available at the Open Connectome Project. In 2012, a Citizen science project called EyeWire began attempting to crowdsource the mapping of the connectome through an interactive game. Independently, important topologies of functional interactions among several hundred cells are also gradually going to be declared (Shimono and Beggs, 2014). Scaling up ultrastructural circuit mapping to the whole mouse brain is currently underway (Mikula, 2012). An alternative approach to mapping connectivity was recently proposed by Zador and colleagues (Zador et al., 2012). Zador's technique, called BOINC (barcoding of individual neuronal connections) uses high-throughput sequencing to map neural circuits. Briefly, the approach consists of (1) labelling each neuron with a unique DNA barcode; (2) transferring barcodes between synaptically coupled neurons (for example using PRV); and (3) fusion of barcodes to represent a synaptic pair. This approach has the potential to be cheap, fast, and extremely high-throughput.

***Tags***: Neuroscience

#### See also
[Functional connectivity](/notes/functional_connectivity), [Effective connectivity](/notes/effective_connectivity), [Connectomics](/notes/connectomics)


## Papers
* Sporns, O. (2013). [The human connectome: origins and challenges](http://www.naturalezacienciaysociedad.org/wp-content/uploads/2013/09/connectome-9.pdf). Neuroimage, 80, 53-61.
* Sporns, O., Tononi, G., & KÃ¶tter, R. (2005). [The human connectome: a structural description of the human brain](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0010042). PLoS Comput Biol, 1(4), e42.
* Van Essen, D. C., & Ugurbil, K. (2012). [The future of the human connectome](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3350760/). Neuroimage, 62(2), 1299-1310.
* Sporns, O. (2011). [The human connectome: a complex network](https://www.researchgate.net/profile/Olaf_Sporns/publication/49770658_The_human_connectome_A_complex_network/links/0fcfd51095acd0d148000000.pdf). Annals of the New York Academy of Sciences, 1224(1), 109-125.
* Van Essen, David C., et al. (2012). [The Human Connectome Project: a data acquisition perspective](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3606888/). Neuroimage, 62(4), 2222-2231.
* Kaiser, M. (2011). [A tutorial in connectome analysis: topological and spatial features of brain networks](http://arxiv.org/pdf/1105.4705). Neuroimage, 57(3), 892-907.
* Fornito, A., Zalesky, A., & Breakspear, M. (2013). [Graph analysis of the human connectome: promise, progress, and pitfalls](https://minerva-access.unimelb.edu.au/bitstream/handle/11343/44053/Graph%20analysis_NeuroImage.pdf?sequence=1). Neuroimage, 80, 426-444.
* Jarrell, Travis A., et al. (2012). [The connectome of a decision-making neural network](https://www.researchgate.net/profile/David_Hall34/publication/230574089_The_connectome_of_a_decision-making_neural_network/links/54a1b5390cf257a6360376c7.pdf). Science, 337(6093), 437-444.
* Van Den Heuvel, M. P., & Sporns, O. (2011). [Rich-club organization of the human connectome](https://www.jneurosci.org/content/31/44/15775.full). The Journal of neuroscience, 31(44), 15775-15786.
* Marcus, Daniel, et al. (2011). [Informatics and data mining tools and strategies for the human connectome project](http://journal.frontiersin.org/article/10.3389/fninf.2011.00004). Frontiers in neuroinformatics, 5, 4.
* Guye, M., Bettus, G., Bartolomei, F., & Cozzone, P. J. (2010). [Graph theoretical analysis of structural and functional connectivity MRI in normal and pathological brain networks](http://link.springer.com/article/10.1007/s10334-010-0205-z). Magnetic Resonance Materials in Physics, Biology and Medicine, 23(5-6), 409-421.
* de Kwaasteniet, Bart, et al. (2013). [Relation between structural and functional connectivity in major depressive disorder](https://www.researchgate.net/profile/Bart_Kwaasteniet2/publication/235521213_Relation_Between_Structural_and_Functional_Connectivity_in_Major_Depressive_Disorder/links/5677c48608ae0ad265c7e90d.pdf). Biological psychiatry, 74(1), 40-47.

## Books
* Sporns, O. (2010). [Networks of the Brain](https://www.goodreads.com/book/show/9655037-networks-of-the-brain). MIT press.
* Seung, S. (2012). [Connectome: How the brain's wiring makes us who we are](https://www.goodreads.com/book/show/11346470-connectome). Houghton Mifflin Harcourt.
* Sporns, O. (2012). [Discovering the human connectome](https://www.goodreads.com/book/show/15857480-discovering-the-human-connectome). MIT press.
* Leergaard, T. B. (Ed.). (2012). [Mapping the connectome: multi-level analysis of brain connectivity](http://journal.frontiersin.org/article/10.3389/fninf.2012.00014). Frontiers E-books.


